lib/fetching/bot-master.js

```diff
@@ -42,18 +42,18 @@ BotMaster.prototype.reset = function() {
 // Start all bots
 BotMaster.prototype.start = function() {
   var self = this;
-  this.enabled = true;
+  this.isFetching = true;
   this.bots.forEach(function(bot) {
-    self.enableBot(bot);
+    self.enableBot(bot, function() {});
```

Comment:
No need for the empty function. Just because you feel like the Source.get requires a callback(err), you don't need the empty function. Changing the enable and disable function signatures is what probably has broken the tests.